opensearch-service-domain-cdk/cdk.context.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
Which of these must a user fill in, and what is the plan to communicate defaults for the ones they don't fill in?

Reply:
The README mentions that some are required.  If the required ones are not present, what happens?

Reply:
General TS/CDK UX comment - I'm not crazy about having a configuration file that the user fills out alongside other code in the repo.  It's lacking a separation between library code and customization.  Even if that's how CDK typically stages stuff, it still smells.

Reply:
I probably agree with this. Do we want to make any assumptions about default values? I'm okay with say having this `default-values.ts` and maybe having a domain name and version in it but don't know if I would want to put anything else in it